cmd/auth/token_test.go

```diff
@@ -221,3 +222,46 @@ func TestToken_loadToken(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadTokenDoesNotMutatePersistentAuthOpts(t *testing.T) {
```

Comment:
If I understood correctly, this test verifies those 3 lines:
```go
    allOpts := append([]u2m.PersistentAuthOption{}, req.PersistentAuthOpts...)
	allOpts = append(allOpts, u2m.WithOAuthArgument(req.OAuthArgument))
	persistentAuth, err := u2m.NewPersistentAuth(ctx, allOpts...)
```
Is that correct? So it effectively tests the `AcquireToken` function, and just that specific part. I'm not really sure if there's enough value in such separate unit test but I think we can keep it 🤔 

Can we move it to `libs/auth/token_loader_test.go`? Also, shouldn't we move other `loadToken` tests there as well, and keep just a single happy path for the `loadToken` itself? Now the core logic is in the `AcquireToken` and basically that's what the `loadToken` unit tests test.